### ISSUE TITLE
数据集处理

### ISSUE THREAD

**Issue by xyc-henu** [shuxjweb/pixel_sampling#18]
val_set_c = ImageDatasetGcnMask(dataset.query_c + dataset.gallery, val_transforms)
    val_loader_c = DataLoader(
        val_set_c, batch_size=cfg.batch_size_test, shuffle=False, num_workers=num_workers,
        collate_fn=val_collate_gcn_mask
您好，您这里的数据处理为什么要把query跟gallery加在一起？
